actors/common/src/main/java/org/sunbird/learner/actors/bulkupload/BaseBulkUploadActor.java

```diff
@@ -314,4 +327,37 @@ protected void performBatchUpdate(List<BulkUploadProcessTask> records) {
       }
     }
   }
+
+  protected void validateFileHeaderFields(
+      Map<String, Object> req, String[] bulkLocationAllowedFields, Boolean allFieldsMandatory)
+      throws IOException {
+    byte[] fileByteArray = null;
+    if (null != req.get(JsonKey.FILE)) {
+      fileByteArray = (byte[]) req.get(JsonKey.FILE);
+    }
+    CSVReader csvReader = null;
+    Boolean flag = true;
+    String[] csvLine;
+    Integer sequence = 0;
+    try {
+      csvReader = getCsvReader(fileByteArray, ',', '"', 0);
+      while (flag) {
+        csvLine = csvReader.readNext();
+        if (ProjectUtil.isNotEmptyStringArray(csvLine)) {
+          continue;
+        }
+        if (sequence == 0) {
+          csvLine = trimColumnAttributes(csvLine);
+          validateBulkUploadFields(csvLine, bulkLocationAllowedFields, allFieldsMandatory);
+          flag = false;
+        }
+      }
+    } catch (Exception ex) {
+      ProjectLogger.log(
+          "BaseBulkUploadActor :validateFileHeaderFields - error " + ex.getMessage(), ex);
```

Comment:
From:
"BaseBulkUploadActor :validateFileHeaderFields - error "

To:
"BaseBulkUploadActor:validateFileHeaderFields: Exception = "